src/main/resources/sql/municipality/get-all.sql

```diff
@@ -0,0 +1,3 @@
+SELECT *
+FROM municipality m
+INNER JOIN municipality_court mc ON mc.municipality_id = m.municipality_id
```

Comment:
So you are just adding the court_id's to the municipality and not the court objects?